lib/chef/provisioning/chef_provider_action_handler.rb

```diff
@@ -42,16 +42,23 @@ def report_progress(description)
       provider.converge_by description do
         # We already did the action, but we trust whoever told us that they did it.
       end
+      puts "\n"
     end
 
     def performed_action(description)
       provider.converge_by description do
         # We already did the action, but we trust whoever told us that they did it.
       end
+      puts "\n"
```

Comment:
Oh, is converge_by not filtering through the chef event formatter?  Probably the right thing is to make sure it does that.  That sucks :/